Shaders/custom_mat_presets/custom_mat_deferred.frag.glsl

```diff
@@ -48,4 +48,8 @@ void main() {
 	#ifdef _EmissionShaded
 		fragColor[GBUF_IDX_EMISSION] = vec4(emissionCol, 0.0);
 	#endif
+	
+	#ifdef _SSRefraction
+		fragColor[GBUF_IDX_REFRACTION] = vec4(ior, opacity, 0.0, 0.0);
```

Comment:
`ior` and `opacity` are probably undefined in this custom shader.

Reply:
I corrected that.